web/modules/custom/hoeringsportal_audit_log/src/Helpers/ConfigHelper.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace Drupal\hoeringsportal_audit_log\Helpers;
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\hoeringsportal_audit_log\Form\SettingsForm;
+
+/**
+ * Config helper.
+ */
+class ConfigHelper {
+  // Limit where audits can be made. If this is expanded, you probably need to
+  // write some code in both SettingsForm and ControllerListener.
+  private const ENABLED_AUDIT_IDS = ['node', 'user'];
+
+  /**
+   * The module config.
+   *
+   * @var \Drupal\Core\Config\ImmutableConfig
+   */
+  private $moduleConfig;
+
+  /**
+   * Confighelper constructor.
+   *
+   * @param \Drupal\Core\Config\ConfigFactoryInterface $configFactory
+   *   The config factory.
+   */
+  public function __construct(ConfigFactoryInterface $configFactory) {
+    $this->moduleConfig = $configFactory->getEditable(SettingsForm::SETTINGS);
+  }
+
+  /**
+   * Returns enabled audits.
+   */
+  public static function getEnabledAuditIds() {
+
+    return self::ENABLED_AUDIT_IDS;
+
+  }
+
+  /**
+   * Save.
+   */
+  public function saveConfig() {
+    $this->moduleConfig->save();
+  }
+
+  /**
+   * Set configuration.
+   */
+  public function setConfiguration(string $configName, array $config): void {
+    $this->moduleConfig->set($configName, $config);
+  }
+
+  /**
+   * Get configuration.
+   */
+  public function getConfiguration(string $configName): array|string {
+    return $this->moduleConfig->get($configName) ?? [];
+  }
```

Comment:
`Config::get` can return something that's neither a `string` nor an `array` (in general), so you may want to check that type of the config value before returning it (and claiming the type is `array|string`).

Consider using `NULL` to signal “no such configuration” and setting the return type to `mixed`.